### PR TITLE
Re-structure web page and fixed sidebar

### PR DESCRIPTION
**Pull request by makoscafee**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,65 +1,49 @@
...
-  <body>
-    <div id="header_wrap" class="outer">
-      <header class="inner" style="max-width: 900px;">
-        <a id="forkme_banner" href="https://github.com/oppia/oppia">
-          View on GitHub
-        </a>
-        <h1 id="project_title">
-          <a href="https://www.oppia.org" class="anchor">Oppia</a>
-        </h1>
+<head>
```
Comment (seanlip) [inline index.html]: This should be indented by 2, since it's inside the <html> tag in the DOM tree (i.e., it isn't a sibling)

Comment (makoscafee) [inline index.html]: Done!

**Comment by makoscafee** [comment]
@amgowano  please would you mind doing  a review?!.
Thanks!.

**Comment by seanlip** [comment]
@amgowano , @MAKOSCAFEE : Should this be merged?

**Comment by makoscafee** [comment]
This PR is clean  but for mobile user, sidebar is hidden.. so  I don't think if it is ready to be merged. It much depend on the @amgowano changes for it to be ready

**Pull request merged**